src/lib/InputComponent.tsx

```diff
@@ -0,0 +1,280 @@
+import React from "react";
```

Comment:
This file was converted to typescript from `InputComponent.tsx`.  I've attempted to improve typing in this file, but it's still not complete

Reply:
I think you meant it was converted from `InputComponent.js`.
I did a side-by side eyeball comparison and the conversion looks good.